adapters/repos/db/vector/flat/index_test.go

```diff
@@ -38,14 +38,20 @@ import (
 	flatent "github.com/weaviate/weaviate/entities/vectorindex/flat"
 )
 
+const (
+	CompressionNone = "none"
+	CompressionPQ   = "pq"
+	CompressionBQ   = "bq"
```

Comment:
can those be reused from `index.go` (it would be necessary to make them public or change current package to `flat`) instead of creating new ones?

Reply:
not used (replaced by ones from `flat` package). to be removed